iceoryx2-pal/posix/build.rs

```diff
@@ -25,8 +28,17 @@ fn main() {
     println!("cargo:rustc-link-lib=acl");
     println!("cargo:rerun-if-changed=src/c/posix.h");
 
+    if !cc::Build::new().get_compiler().is_like_clang() {
+        println!("cargo:warning=Clang is not available, you may set up environment {} to avoid missing some headers.", IOX2_CLANG_ARG);
```

Comment:
I'm not sure if I would understand what to do when reading this message. Can you be more precise. Maybe with an example.